ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -118,7 +118,7 @@ class AdminWorkspaceImpl extends React.Component<UrlParamsProps, State> {
 
   renderHighChart(cloudStorageTraffic: CloudStorageTraffic): ReactFragment {
     HighCharts.setOptions({
-      global: {
+      time: {
```

Comment:
https://api.highcharts.com/highcharts/global.useUTC:

This option is deprecated since v6.0.5. Instead, use time.useUTC that supports individual time settings per chart.